### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by binaryberry**
Updating README to match [our styleguide](https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md), and in preparation for pentesting.

[Trello](https://trello.com/c/5UuqGR6L/356-document-applications-ready-for-pentest-medium)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,72 +3,93 @@
...
+manages document creation, metadata, slugs, titles, etc. It is sometimes referred to as "mainstream publisher".
 
-## Running in development
+##Screenshots
 
-If you're just interested in running the Publisher locally, with a minimum of interaction
-with other apps, here's how.
-
-### Create a user
+![alt tag](https://raw.githubusercontent.com/alphagov/publisher/update-readme/app/assets/images/publisher_document_screenshot.png)
```
Comment (tijmenb) [inline README.md]: You can make these links relative - that also works:

https://raw.githubusercontent.com/alphagov/content-tagger/master/README.md

Comment (binaryberry) [inline README.md]: Veel dank! :rainbow:

File: README.md [hunk 2]
```diff
@@ -3,72 +3,93 @@
...
-## Asset Manager
+##### Create a user
 
-If your models include media assets such as images and video, you will need to run the asset-manager
-app alongside publisher.
+This step is if you haven't [imported production data](https://github.gds/gds/development#8-import-production-data).
 
-See the [asset-manager](http://github.com/alphagov/asset-manager) project for app-specific setup 
-instructions.
+publisher$ rails console
```
Comment (boffbowsh) [inline README.md]: Put these in a multi-line code block but without the prefix, then people can copy and paste.

File: README.md [hunk 3]
```diff
@@ -3,72 +3,93 @@
...
 
-This application uses [statsd-ruby](http://rubygems.org/gems/statsd-ruby) to send metrics to
-[statsd](https://github.com/etsy/statsd/). If a statsd process isn't present on the server
-it won't matter as statsd-ruby sends metrics over UDP. If a statsd process is present then
-it'll send strings with the respective increment/decrement/gauge function to use.
+If you're just interested in running the Publisher locally, with a minimum of interaction
+with other apps, here's how.
 
-## Asset Manager
+##### Create a user
```
Comment (boffbowsh) [inline README.md]: Maybe just 4 #s?

File: README.md (hunk 1 above)
Comment (tijmenb) [inline README.md]: Geen probleem!

File: README.md (hunk 2 above)
Comment (binaryberry) [inline README.md]: Thanks, addressed now!

File: README.md (hunk 3 above)
Comment (binaryberry) [inline README.md]: Thanks, addressed now!

File: README.md [hunk 4]
```diff
@@ -3,72 +3,96 @@
...
-They can be imported using a rake task:
+```shell
+rake api_clients:create[publisher,publisher@example.com,asset-manager,signin]
+```
 
-    bundle exec rake local_transactions:import SOURCE=/path/to/local_authority_service_details.CSV
+### Running the application
 
-## Statsd
+If you're just interested in running the Publisher locally, with a minimum of interaction
```
Comment (boffbowsh) [inline README.md]: I think this whole section should be replaced with VM-specific instructions, as this is our prescribed development workflow.

File: README.md [hunk 5]
```diff
@@ -3,72 +3,96 @@
...
-    publisher$ PANOPTICON_URI="http://localhost:3001" bundle exec rails server
-
-## Local Transactions
-
-There is no UI or automated process for importing the source data for local transactions.
+manages document creation, metadata, slugs, titles, etc. It is sometimes referred to as "mainstream publisher".
+
+##Screenshots
+
+![alt tag](app/assets/images/publisher_document_screenshot.png)
```
Comment (boffbowsh) [inline README.md]: These images should go in the `doc/` directory so they don't become part of the application's asset pipeline

File: README.md (hunk 4 above)
Comment (binaryberry) [inline README.md]: Thanks, addressed now!

File: README.md (hunk 5 above)
Comment (binaryberry) [inline README.md]: Thanks! Addressed now

**Comment by binaryberry** [comment]
@boffbowsh this should be ready for another review, I believe I've addressed all comments. I squashed your commit into mine, hope it's ok!

**Comment by boffbowsh** [comment]
Made a few changes myself 👍

**Pull request merged**